### ISSUE TITLE
Exp2: Batch size testing

### ISSUE THREAD

**Issue by Micky774** [quinngroup/CiliaRepresentation#5]
Need to test the effect of batch size on learning. Specifically, we may want to focus on how batch size interacts with the learning rates suggested by exp1

**Comment by Micky774** [comment]
Adopting a default learning rate of 1e-5, we will test batch sizes: {10,20,40,80} over 30 epochs each

**Comment by AnonymousFool** [comment]
Pushed the bash script. Ready to go.

**Comment by Micky774** [comment]
Initial impressions lead me to believe that there's enough variance in the underlying data set that we don't necessarily need to exploit low batch sizes for induced variance and gradient noise. I think, especially with the full data set, we'll just want as large of a batch size as we can get. Follow up to this once we run Exp7. Also, worth noting that we shouldn't benchmark them based on fixed epoch count, but rather a fixed time, since a smaller batch size naturally implies _more_ weight updates in the same epoch count, but that doesn't necessarily translate to best for training overall, since our actual concern is efficiency over time.

Initial conclusions: batch size 80 per card is recommended. Further testing in Exp7 recommended.
